pages/api/playlists/index.ts

```diff
@@ -0,0 +1,22 @@
+import { getPlaylists } from '@/lib/spotify';
+import { isBadStatusCode } from '@/lib/utils';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getSession } from 'next-auth/client';
+
+type NextAPIRequestWithId = NextApiRequest;
+
+export default async (req: NextAPIRequestWithId, res: NextApiResponse) => {
```

Comment:
You can remove the type redeclaration on line 8 and change this to (since there is no `id` being passed into the params):

```suggestion
export default async (req: NextAPIRequest, res: NextApiResponse) => {
```